front_end/handlers/BaseUserHandler.py

```diff
@@ -0,0 +1,58 @@
+from helper import *
```

Comment:
If I understand correctly, the classes that inherit from this class should not need to import these libraries because they are imported in BaseUserHandler. You should only need to import from tornado.web and any other modules that are specific to each handler.